script.js

```diff
@@ -249,15 +278,61 @@ function handleLocalitiesLayer(geojson) {
   });
 
   // Add data to the map
-  localitiesLayer.addTo(map);
+  citiesLayer.addTo(map);
 
-  // Move the map view so that the localitiesLayer is visible
-  map.fitBounds(localitiesLayer.getBounds(), {
+  // Move the map view so that the citiesLayer is visible
+  map.fitBounds(citiesLayer.getBounds(), {
     paddingTopLeft: [12, 120],
     paddingBottomRight: [12, 12]
   });
 
-  return localitiesLayer;
+  return citiesLayer;
+}
+
+function handleCountiesLayer(geojson) {
+  const layerOptions = {
+    style: feature => {
+      // style states based on whether their moratorium has passed
+      if (feature.properties.passed === true) {
+        return {
+          color: "#4dac26",
+          fillColor: "#b8e186",
+          fillOpacity: fillOpacity,
+          weight: strokeWeight
+        };
+      } else if (feature.properties.passed === false) {
+        return {
+          color: "#d01c8b",
+          fillColor: "#f1b6da",
+          fillOpacity: fillOpacity,
+          weight: strokeWeight
+        };
+      } else {
```

Comment:
@timhitchins not sure this else condition will be reached if the changes are made above, so then you could just keep it to:
```js
if (feature.properties.passed) {
  // first style
} else {
  // second style
}
```